app/main.py

```diff
@@ -1,3 +1,75 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> "Distance":
+        if isinstance(other, Distance):
+            self.km = self.km + other.km
+            return Distance(
+                self.km
+            )
+        else:
+            self.km = self.km + other.real
```

Comment:
The line `self.km = self.km + other.real` assumes that `other` has a `real` attribute, which is not the case for integers. You should directly add `other` to `self.km` if `other` is an integer.